lithops/standalone/standalone.py

```diff
@@ -183,13 +184,18 @@ def invoke(self, job_payload):
 
         def create_workers(workers_to_create):
             current_workers_old = set(self.backend.workers)
+            futures = []
             with cf.ThreadPoolExecutor(workers_to_create+1) as ex:
-                ex.submit(lambda: self.backend.master.create(check_if_exists=True)
-                          if not self._is_master_service_ready() else False)
+                if not self._is_master_service_ready():
+                    futures.append(ex.submit(lambda: self.backend.master.create(check_if_exists=True)))
```

Comment:
The idea of putting` _is_master_service_ready()` within the thread was to speedup the creation process. This method outside will slowdown it. Apart of this, some days ago I ran some tests and I saw that it is not needed to save the future into `futures` since thanks to the way the function is submitted, it already raises the exceptions (if any) without having to call `future.result()`.

Reply:
lease, consider the following scenario:
`self._is_master_service_ready()` inside the thread and not checking if that thread raised exception.
`self._is_master_service_ready` calls dismantle + raises `LithopsValidationError`, but the exception is not raised, then flow continues to create N workers without any purpose as their master is unusable.
And it stops and raises only on the next `self._is_master_service_ready()` leaving us with N workers to be wasted.

Reply:
What I am saying is that in this particular case, the exceptions are correctly raised even if you put this method within the thread, and there is no need to call future.result(). You can verify it by putting a temporary `raise Exception('TEST')` in the first line of the `self._is_master_service_ready()` method.

Reply:
@JosepSampe That is exactly what i did and nothing been raised

Reply:
Very weird, in my case it raises the exception

Reply:
@JosepSampe It raises the exception without creating any workers? Are you sure it raises it there and not seevral lines later when it checks `_is_master_service_ready` again out of the thread?

Reply:
This is what exactly I though, because the correct approach would be the one you put here, but I put prints everywhere and for some reason it raises the exception in the thread. If you don't see the same behavior as me, it is better to keep your changes.